### PR TITLE
issue session cookie doesn't expire

### PR DESCRIPTION
**Pull request by OmarTariq612**
Not calling the function makes the created cookie a session cookie instead of making it last for a month.

image

Screenshot from 2023-05-31 01-04-55

image

### CODE REVIEW

**Review by Mahmoudgalalz — CHANGES_REQUESTED** [review]
thanks for this @OmarTariq612 I just want to add an additional change to that, to avoid making this uncompiled in the client's code

removing the directive `is:inline` will show errors in the IDE as it will think that the DOM API isn't available it's a known issue in Astro within TS configs, and needs an update to allowJs If you could do that too will be a great addition, 

so for merging it it should be complete the needs as will
- remove is:inline
- add allowJs in ts config

**Comment by OmarTariq612** [comment]
I removed the `is:inline` directive but I didn't have errors calling the DOM API, the only errors I faced are related to types so I fixed them, also I noticed that `cookie` and `url` constants aren't being used so I removed them.

File: src/pages/newjob/index.astro
Comment (Mahmoudgalalz) [inline src/pages/newjob/index.astro:1]: This is awesome @OmarTariq612 Thanks for the non-noticeable bug

**Review by Mahmoudgalalz — APPROVED** [review]

**Comment by Mahmoudgalalz** [comment]
we hope you would like to join us on [Slack](https://join.slack.com/t/juniorjobs/shared_invite/zt-1t3koh4xe-HqiPVA7umjpZ4O8EjEpFzg)

to discuss and be involved in early contribution, before adding them in the repo

**Pull request merged**